build/yamls/antrea.yml

```diff
@@ -5691,8 +5691,7 @@ data:
 
     nodePortLocal:
     # Enable NodePortLocal, a feature used to make Pods reachable using port forwarding on the host. To
-    # enable this feature, you need to set "enable" to true, and ensure that the NodePortLocal feature
-    # gate is also enabled (which is the default).
+    # enable this feature, you need to set "enable" to true.
```

Comment:
Since it's GA now, I think the default should be `true`? @tnqn @antoninbas

Reply:
Change the default value to `true` now.

Reply:
Thanks for @tnqn 's comments. I set the default value to `false` now.